distribution/helm/vector-aggregator/templates/haproxy/_helpers.tpl

```diff
@@ -0,0 +1,71 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
I have a mixed feeling about that as some template have it and some don't, but that's not really a problem per se.

Reply:
🤔 yeah, I think this was copied from the `templates/_helper.tpl` file in the aggregator. No strong feelings on it.